File: Algorithms/Warmup/Mini-Max Sum.java
/*
* Contributer : github.com/shikhar-07
* Email : shikharanandg@gmail.com
*/
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

  public static void main(String[] args) {
		int i;
		long sum = 0;
		long  max = 0;
		Long  min = (long) Math.pow(10,10);
		Scanner input = new Scanner(System.in);
		long [] array = new long[5];
		long [] temp = new long[5];
		for(i = 0 ; i < 5 ; i++)
		{
			array[i] = input.nextLong();
			sum+=array[i];
		}
		for(i = 0 ; i < 5 ; i++)
		{
			temp[i] = sum - array[i];
			
			if(temp[i] > max)
			{
				max = temp[i];
			}
			if(temp[i] < min)
			{
				min = temp[i];
			}
		}
		System.out.print(min);
		System.out.print(" ");
		System.out.print(max);
  }		
}	
		
		

	

File: Algorithms/Warmup/Mini-Max Sum.cpp
/*
   Contributer : github.com/jaswal72
   Email : shubhamjaswal772@gmail.com
*/
#include <map>
#include <set>
#include <list>
#include <cmath>
#include <ctime>
#include <deque>
#include <queue>
#include <stack>
#include <string>
#include <bitset>
#include <cstdio>
#include <limits>
#include <vector>
#include <climits>
#include <cstring>
#include <cstdlib>
#include <fstream>
#include <numeric>
#include <sstream>
#include <iostream>
#include <algorithm>
#include <unordered_map>

using namespace std;


int main(){
    long int a[5],max=0,min=1000000000000,sum=0;
    for(int i=0;i<5;i++){
        cin>>a[i];
        if(a[i]>max)
            max=a[i];
        if(a[i]<min)
            min=a[i];
        sum+=a[i];
    }
    cout<<sum-max<<" "<<sum-min;
    return 0;
}
